examples/layers/JSONLayers/Administrative.json

```diff
@@ -1,14 +1,14 @@
 {
   "id": "Administrative",
   "title": "Administrative delimitations",
-  "visible": false,
+  "visible": true,
```

Comment:
could you keep false?

Reply:
No, I think we can loose time with visible = false...